pkg/sqlite/load.go

```diff
@@ -77,7 +79,21 @@ func (s *SQLLoader) AddOperatorBundle(bundle *registry.Bundle) error {
 		return fmt.Errorf("csv name not found")
 	}
 
-	if _, err := stmt.Exec(csvName, csvBytes, bundleBytes, bundleImage); err != nil {
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		return err
+	}
+
+	skiprange, ok := csv.Annotations[SkipRangeAnnotationKey]
+	if !ok {
+		skiprange = ""
+	}
+	version, err := csv.GetVersion()
+	if err != nil {
+		version = ""
+	}
```

Comment:
It looks like we're doing this in multiple places. Can we surface these as methods of `registry.Bundle()` and memoize this work?